CMakeLists.txt

```diff
@@ -194,6 +208,11 @@ include(cmake/package_helpers.cmake)
 ginkgo_find_package(GTest "GTest::GTest;GTest::Main" FALSE 1.8.1)
 ginkgo_find_package(gflags gflags FALSE 2.2.2)
 ginkgo_find_package(RapidJSON rapidjson TRUE 1.1.0)
+if(NOT GINKGO_FORCE_NO_HWLOC AND GINKGO_HAVE_HWLOC)
+  ginkgo_find_package(HWLOC hwloc FALSE 2.1)
+endif()
+set(GKO_HWLOC_XMLFILE "${HWLOC_XMLFILE}")
+message(STATUS "HWLOC XML file set to: ${GKO_HWLOC_XMLFILE}")
```

Comment:
Shouldn't it be `$ENV{HWLOC_XMLFILE}`? Also, isn't this unneeded since it's standard hwloc anyway?